pkg/api/v1beta3/dynakube/activegate/props.go

```diff
@@ -92,20 +101,37 @@ func (ag *Spec) IsMetricsIngestEnabled() bool {
 	return ag.IsMode(MetricsIngestCapability.DisplayName)
 }
 
+func (ag *Spec) IsAutomaticTlsSecretEnabled() bool {
+	return ag.TlsSecretName == "" && ag.trustedCAs != "" && ag.automaticTLSCertificateEnabled
```

Comment:
why do you need `trustedCAs` for creating the automatic TLS cert creation?

Reply:
`trustedCAs` cause OAs to be picky about certificates, that's when the use of AG certificates becomes mandatory for OA communication to work. If no trustedCAs are used, we probably still could create the certs, but it would also work without.

Reply:
let's not tie this to `trustedCAs` with no other way to use the automatic tls cert.

Reply:
logic changed. `IsAutomaticTlsSecretEnabled()` doesn't depend on TrustedCAs.